backend/server/api/chat_models.py

```diff
@@ -0,0 +1,18 @@
+"""Endpoint for listing available chat models for extraction."""
+from typing import List, get_args
+
+from fastapi import APIRouter
+
+from server.settings import ModelNameLiteral
+
+router = APIRouter(
+    prefix="/chat_models",
+    tags=["Available chat models"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.get("")
+def list_chat_models() -> List[str]:
```

Comment:
I suspect we'll want to generalize this a bit. There might be other configuration that we'll want to fetch (e.g., does the server support retrieving content by URL)

Reply:
We can do an end point like this:

```
def get_configurables():
   return {
     'models': []

   }
```

Reply:
thanks, makes sense